backend/coreapp/compilers.py

```diff
@@ -854,6 +854,8 @@ def preset_from_name(name: str) -> Optional[Preset]:
         "-O2 -g3 -mips2 -woff 624",
         diff_flags=["-Mreg-names=32"],
     ),
+    
+    Preset("AeroGauge", IDO53, "-O2 -g0 -mips2 -32 "),
```

Comment:
```suggestion
    Preset("AeroGauge", IDO53, "-O2 -mips2"),
```
`-32` is passed by `IDO53` already, `-g0` is the default so it's fine to leave off.